### PR TITLE
Task: base entity

### PR DESCRIPTION
**Pull request by DangHuan2k3**
## Description

- Add entities for this project base database built
- Edit UserEntity to User
- Edit .prettierrc for format: tabwith 
- Define enum constants for entity fields

## Checklist

- [x] Code follows the project’s code style guidelines.
- [x] Verified all related functionalities.
- [x] Updated documentation (if needed).

## Screenshots (if any)

database on local

## Additional Information

- Any other information you want the reviewer to know
[one-to-one relations (typeorm)](https://orkhan.gitbook.io/typeorm/docs/one-to-one-relations)
[one-to-many relations (typeorm)](https://orkhan.gitbook.io/typeorm/docs/many-to-one-one-to-many-relations)
[many-to-many with custom properties (typeorm)](https://orkhan.gitbook.io/typeorm/docs/many-to-many-relations#many-to-many-relations-with-custom-properties)

### CODE REVIEW

File: src/entities/result.entity.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    type: 'date',
+    nullable: false
+  })
+  public timeStart: Date;
+
+  @Column({
+    type: 'interval',
+    nullable: false
+  })
+  public timeTaken: number;
```
Comment (nguyennhatninh) [inline src/entities/result.entity.ts:26]: should use type ''string''

**Review by nguyennhatninh — COMMENTED** [review]
Should use type ''string'' at line 26, file src/entities/result.entities.ts

**Review by huywzz — APPROVED** [review]
Looks good to me

**Comment by DangHuan2k3** [comment]
Sorry, I make some mistake and improve my code. I close this PR.

**Pull request closed without merging**